.github/workflows/release.yaml

```
@@ -465,104 +465,48 @@ jobs:
       - name: Download data
         run: sh download_data.sh
 
-      - name: Install s3cmd
+      - name: Install s3cmd (staging)
         if: github.ref != 'refs/heads/release'
         uses: ./.ci/install-s3cmd
         with:
-          access-key: ${{ secrets.STAGING_ASSETS_BUCKET_ACCESS_KEY }}
-          secret-key: ${{ secrets.STAGING_ASSETS_BUCKET_SECRET_KEY }}
+          access-key: ${{ secrets.AI_ASSETS_STG_BUCKET_ACCESS_KEY }}
+          secret-key: ${{ secrets.AI_ASSETS_STG_BUCKET_SECRET_KEY }}
+
+      - name: Install s3cmd (production)
+        if: github.ref = 'refs/heads/release'
```

just a reminder to all of us, we should actually run stuff before we merge it to see [if it really works](https://github.com/xaynetwork/xayn_ai/actions/runs/1217825888/workflow#L476) :/
```suggestion
        if: github.ref == 'refs/heads/release'
```
it's going to be fixed in the next pr #202